src/applications/disability-benefits/2346/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "appName": "Form-2346",
+  "entryFile": "./app-entry.jsx",
+  "entryName": "mdt-2346",
+  "rootUrl": "/mdt/2346",
```

Comment:
It's your team's call, but I personally steer away from acronyms and go with the fully verbose name to add the extra meaning. Would this be `medical-device-something`? Or maybe `batteries-and-accessories`?

Reply:
We will leave this to business.

Reply:
I think it'd be better to do it now, instead of waiting.

Reply:
I agree with @Mottie and @ncksllvn

Reply:
Done